### PR TITLE
Remove unnecessary usage of gomega libraries

### PR DESCRIPTION
**Pull request by willbeason**
Wonderfully, this reduces our imported library code by ~180,000 lines.

### CODE REVIEW

File: constraint/pkg/apis/templates/v1/constrainttemplate_types_test.go [hunk 1]
```diff
@@ -155,22 +181,22 @@ func TestValidationVersionConversionAndTransformation(t *testing.T) {
 		{
 			name: "Two deep properties, LegacySchema=true",
 			v: &Validation{
-				LegacySchema:    pointer.Bool(true),
```
Comment (julianKatz) [inline constraint/pkg/apis/templates/v1/constrainttemplate_types_test.go:158]: I feel like `pointer.Bool(true)` is a better tool, but the difference is minimal.

**Review by julianKatz — APPROVED** [review]

File: constraint/pkg/apis/templates/v1/constrainttemplate_types_test.go (hunk 1 above)
Comment (willbeason) [reply constraint/pkg/apis/templates/v1/constrainttemplate_types_test.go:158]: This was a merge mistake

**Review by maxsmythe — APPROVED** [review]
LGTM

**Comment by julianKatz** [comment]
ping :) @ritazh @sozercan 

Will is going to be out for a bit, so I'll try to get this merged while he's away.  Would be annoying for him if the code changes under him while he's out.

**Comment by willbeason** [comment]
@ritazh ? @sozercan ?

**Pull request merged**